main.go

```diff
@@ -33,11 +41,114 @@ func main() {
 	sync := marathon.NewMarathonSync(remote, consul)
 	go sync.Sync()
 
-	// set up routes
+	fh := &ForwardHandler{consul}
+
+	v, err := remote.Version()
+	if err != nil {
+		log.WithError(err).Warn("version parsing failed, assuming >= 0.9.0")
+		v, _ = version.NewVersion("0.9.0")
+	}
+	minVersion, _ := version.NewConstraint(">= 0.9.0")
+
+	if minVersion.Check(v) {
+		log.Info(fmt.Sprintf("detected Marathon v%s with /v2/events endpoint", v))
```

Comment:
The rest of the project uses the convention of separating fields out into separate fields in logrus calls. So this would be `log.WithField("version", v).Info("detected Marathon with /v2/events endpoint")`. The one below could change too.